### PR TITLE
Create model

### PR DESCRIPTION
**Pull request by dinhthai2206**

### CODE REVIEW

File: test/models/theme_test.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (manhcuongdtbk) [inline test/models/theme_test.rb]: Change ‘ ‘ to “ “

File: test/models/speech_theme_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (manhcuongdtbk) [inline test/models/speech_theme_test.rb]: Change ‘ ‘ to “ “

File: test/models/speech_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (manhcuongdtbk) [inline test/models/speech_test.rb]: Change ‘ ‘ to “ “

File: test/models/location_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (manhcuongdtbk) [inline test/models/location_test.rb]: Change ‘ ‘ to “ “

File: app/models/speech.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class Speech < ApplicationRecord
+  belongs_to :location
+  belongs_to :user
+  has_many :speech_themes
+  has_many :themes, through: :speech_themes
+
+  validates :title, presence: true
+  validates :content, presence: true
+  validates :speaking_day, presence: true
+  validates :location_id, presence: true
```
Comment (manhcuongdtbk) [inline app/models/speech.rb]: Add validate user_id

File: app/models/speech.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+class Speech < ApplicationRecord
+  belongs_to :location
+  belongs_to :user
+  has_many :speech_themes
+  has_many :themes, through: :speech_themes
+
+  validates :title, presence: true
```
Comment (chintk) [inline app/models/speech.rb]: validates :title, :content, :speaking_day, :location_id, :user_id, presence: true

**Pull request merged**